sunpy/map/sources/suvi.py

```diff
@@ -0,0 +1,59 @@
+"""SUVI Map subclass definitions"""
+from __future__ import absolute_import, print_function, division
+#pylint: disable=W0221,W0222,E1101,E1121
+
+__author__ = "Jack Ireland"
+__email__ = "jack.ireland@nasa.gov"
+
+import matplotlib.pyplot as plt
+
+from astropy.visualization.mpl_normalize import ImageNormalize
+from astropy.visualization import AsinhStretch
+
+from sunpy.map import GenericMap
+from sunpy.map.sources.source_type import source_stretch
+
+__all__ = ['SUVIMap']
+
+
+class SUVIMap(GenericMap):
+    """SUVI Image Map.
```

Comment:
This doc string should be improved. It should mention that it is on GOES. it is not necessary to state that it replaces SXI. It should state explicitly how many channels it supports and not refer to color tables or compare itself to AIA. It could also give other parameters such as field of view, cadence, and angular resolution. The reference list is also lacking. It should at least refer to the instrument paper. I always refer to the AIAmap which is a good example of a map source header.

Reply:
from @hayesla instrument paper http://doi.org/10.1007/s11207-019-1411-0

Reply:
That is not an appropriate paper to reference (to my reading).  It's a paper about using SUVI to image the extended corona, and doesn't go into a lot of detail about the instrument and how it is nominally operated.

Reply:
I'd argue that this reference describes the instrument better: https://doi.org/10.3847/2041-8213/aaa28e